internal/pkg/manifest/backend_svc_test.go

```diff
@@ -259,7 +259,7 @@ func TestBackendSvc_ApplyEnv(t *testing.T) {
 				TaskConfig: TaskConfig{
 					Count: Count{
 						Autoscaling: Autoscaling{
-							CPU: aws.String("70%"),
+							CPU: percentageP("70%"),
```

Comment:
will this also work correctly if there is no % sign specified?

Reply:
yeah since we just trim the suffix "%" off but I can add a test case for this!